cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/sw.js

```diff
@@ -0,0 +1,70 @@
+workbox.setConfig({ debug: true });
```

Comment:
If you don't need offline functionality, I'd remove this file and tweak the webpack config to stop looking for it.

Reply:
I'd like to leave it in for the time being until other decisions are made.